src/Services/StoreClient.php

```diff
@@ -216,6 +216,7 @@ public function createInAppPurchaseCart(string $extensionName, string $feature,
                     'json' => [
                         'extensionName' => $extensionName,
                         'inAppFeatureIdentifier' => $feature,
+                        'variant' => $variant,
```

Comment:
is the sbp prepared to receive an empty string instead of null or not set at all?